src/osd/ReplicatedBackend.cc

```diff
@@ -1010,7 +1010,7 @@ void ReplicatedBackend::issue_op(
       set<pg_shard_t> replicas = parent->get_acting_recovery_backfill_shards();
       replicas.erase(parent->whoami_shard());
       ss << "waiting for subops from " << replicas;
-      op->op->mark_sub_op_sent(ss.str());
+      op->op->mark_sub_op_sent(ss.str().c_str());
```

Comment:
This is not actually correct -- c_str() returns a pointer to dynamically allocated memory which is freed when the temporary string returned from stringstream::str() is destructed.

Reply:
I got it